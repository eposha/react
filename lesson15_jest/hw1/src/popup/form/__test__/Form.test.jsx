import React from "react";
import { shallow } from "enzyme";
import Form from "../Form";

describe("<Form/>", () => {
  it("should display Form", () => {
    const wrappedComponent = shallow(<Form />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});
