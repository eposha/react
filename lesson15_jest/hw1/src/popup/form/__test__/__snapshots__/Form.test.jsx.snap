// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form/> should display Form 1`] = `
<Fragment>
  <form
    className="popup__form"
    onSubmit={[Function]}
  >
    <input
      className="name-event"
      name="nameEvent"
      onChange={[Function]}
      placeholder="Event name"
      type="text"
    />
    <div
      className="date-wrapper"
    >
      <i
        className="Tiny material-icons"
      >
        access_time
      </i>
      <input
        className="date-event begin"
        name="dateEvent"
        onChange={[Function]}
        type="date"
      />
      <select
        className="bgn-time select-time"
        name="timeEvent"
        onChange={[Function]}
      >
        <option
          key="0"
          value="00:00"
        >
          00:00
        </option>
        <option
          key="1"
          value="00:15"
        >
          00:15
        </option>
        <option
          key="2"
          value="00:30"
        >
          00:30
        </option>
        <option
          key="3"
          value="00:45"
        >
          00:45
        </option>
        <option
          key="1"
          value="01:00"
        >
          01:00
        </option>
        <option
          key="2"
          value="01:15"
        >
          01:15
        </option>
        <option
          key="3"
          value="01:30"
        >
          01:30
        </option>
        <option
          key="4"
          value="01:45"
        >
          01:45
        </option>
        <option
          key="2"
          value="02:00"
        >
          02:00
        </option>
        <option
          key="3"
          value="02:15"
        >
          02:15
        </option>
        <option
          key="4"
          value="02:30"
        >
          02:30
        </option>
        <option
          key="5"
          value="02:45"
        >
          02:45
        </option>
        <option
          key="3"
          value="03:00"
        >
          03:00
        </option>
        <option
          key="4"
          value="03:15"
        >
          03:15
        </option>
        <option
          key="5"
          value="03:30"
        >
          03:30
        </option>
        <option
          key="6"
          value="03:45"
        >
          03:45
        </option>
        <option
          key="4"
          value="04:00"
        >
          04:00
        </option>
        <option
          key="5"
          value="04:15"
        >
          04:15
        </option>
        <option
          key="6"
          value="04:30"
        >
          04:30
        </option>
        <option
          key="7"
          value="04:45"
        >
          04:45
        </option>
        <option
          key="5"
          value="05:00"
        >
          05:00
        </option>
        <option
          key="6"
          value="05:15"
        >
          05:15
        </option>
        <option
          key="7"
          value="05:30"
        >
          05:30
        </option>
        <option
          key="8"
          value="05:45"
        >
          05:45
        </option>
        <option
          key="6"
          value="06:00"
        >
          06:00
        </option>
        <option
          key="7"
          value="06:15"
        >
          06:15
        </option>
        <option
          key="8"
          value="06:30"
        >
          06:30
        </option>
        <option
          key="9"
          value="06:45"
        >
          06:45
        </option>
        <option
          key="7"
          value="07:00"
        >
          07:00
        </option>
        <option
          key="8"
          value="07:15"
        >
          07:15
        </option>
        <option
          key="9"
          value="07:30"
        >
          07:30
        </option>
        <option
          key="10"
          value="07:45"
        >
          07:45
        </option>
        <option
          key="8"
          value="08:00"
        >
          08:00
        </option>
        <option
          key="9"
          value="08:15"
        >
          08:15
        </option>
        <option
          key="10"
          value="08:30"
        >
          08:30
        </option>
        <option
          key="11"
          value="08:45"
        >
          08:45
        </option>
        <option
          key="9"
          value="09:00"
        >
          09:00
        </option>
        <option
          key="10"
          value="09:15"
        >
          09:15
        </option>
        <option
          key="11"
          value="09:30"
        >
          09:30
        </option>
        <option
          key="12"
          value="09:45"
        >
          09:45
        </option>
        <option
          key="10"
          value="10:00"
        >
          10:00
        </option>
        <option
          key="11"
          value="10:15"
        >
          10:15
        </option>
        <option
          key="12"
          value="10:30"
        >
          10:30
        </option>
        <option
          key="13"
          value="10:45"
        >
          10:45
        </option>
        <option
          key="11"
          value="11:00"
        >
          11:00
        </option>
        <option
          key="12"
          value="11:15"
        >
          11:15
        </option>
        <option
          key="13"
          value="11:30"
        >
          11:30
        </option>
        <option
          key="14"
          value="11:45"
        >
          11:45
        </option>
        <option
          key="12"
          value="12:00"
        >
          12:00
        </option>
        <option
          key="13"
          value="12:15"
        >
          12:15
        </option>
        <option
          key="14"
          value="12:30"
        >
          12:30
        </option>
        <option
          key="15"
          value="12:45"
        >
          12:45
        </option>
        <option
          key="13"
          value="13:00"
        >
          13:00
        </option>
        <option
          key="14"
          value="13:15"
        >
          13:15
        </option>
        <option
          key="15"
          value="13:30"
        >
          13:30
        </option>
        <option
          key="16"
          value="13:45"
        >
          13:45
        </option>
        <option
          key="14"
          value="14:00"
        >
          14:00
        </option>
        <option
          key="15"
          value="14:15"
        >
          14:15
        </option>
        <option
          key="16"
          value="14:30"
        >
          14:30
        </option>
        <option
          key="17"
          value="14:45"
        >
          14:45
        </option>
        <option
          key="15"
          value="15:00"
        >
          15:00
        </option>
        <option
          key="16"
          value="15:15"
        >
          15:15
        </option>
        <option
          key="17"
          value="15:30"
        >
          15:30
        </option>
        <option
          key="18"
          value="15:45"
        >
          15:45
        </option>
        <option
          key="16"
          value="16:00"
        >
          16:00
        </option>
        <option
          key="17"
          value="16:15"
        >
          16:15
        </option>
        <option
          key="18"
          value="16:30"
        >
          16:30
        </option>
        <option
          key="19"
          value="16:45"
        >
          16:45
        </option>
        <option
          key="17"
          value="17:00"
        >
          17:00
        </option>
        <option
          key="18"
          value="17:15"
        >
          17:15
        </option>
        <option
          key="19"
          value="17:30"
        >
          17:30
        </option>
        <option
          key="20"
          value="17:45"
        >
          17:45
        </option>
        <option
          key="18"
          value="18:00"
        >
          18:00
        </option>
        <option
          key="19"
          value="18:15"
        >
          18:15
        </option>
        <option
          key="20"
          value="18:30"
        >
          18:30
        </option>
        <option
          key="21"
          value="18:45"
        >
          18:45
        </option>
        <option
          key="19"
          value="19:00"
        >
          19:00
        </option>
        <option
          key="20"
          value="19:15"
        >
          19:15
        </option>
        <option
          key="21"
          value="19:30"
        >
          19:30
        </option>
        <option
          key="22"
          value="19:45"
        >
          19:45
        </option>
        <option
          key="20"
          value="20:00"
        >
          20:00
        </option>
        <option
          key="21"
          value="20:15"
        >
          20:15
        </option>
        <option
          key="22"
          value="20:30"
        >
          20:30
        </option>
        <option
          key="23"
          value="20:45"
        >
          20:45
        </option>
        <option
          key="21"
          value="21:00"
        >
          21:00
        </option>
        <option
          key="22"
          value="21:15"
        >
          21:15
        </option>
        <option
          key="23"
          value="21:30"
        >
          21:30
        </option>
        <option
          key="24"
          value="21:45"
        >
          21:45
        </option>
        <option
          key="22"
          value="22:00"
        >
          22:00
        </option>
        <option
          key="23"
          value="22:15"
        >
          22:15
        </option>
        <option
          key="24"
          value="22:30"
        >
          22:30
        </option>
        <option
          key="25"
          value="22:45"
        >
          22:45
        </option>
        <option
          key="23"
          value="23:00"
        >
          23:00
        </option>
        <option
          key="24"
          value="23:15"
        >
          23:15
        </option>
        <option
          key="25"
          value="23:30"
        >
          23:30
        </option>
        <option
          key="26"
          value="23:45"
        >
          23:45
        </option>
      </select>
      <span>
        -
      </span>
      <select
        className="bgn-end select-time"
        name="endTimeEvent"
        onChange={[Function]}
      >
        <option
          key="0"
          value="00:00"
        >
          00:00
        </option>
        <option
          key="1"
          value="00:15"
        >
          00:15
        </option>
        <option
          key="2"
          value="00:30"
        >
          00:30
        </option>
        <option
          key="3"
          value="00:45"
        >
          00:45
        </option>
        <option
          key="1"
          value="01:00"
        >
          01:00
        </option>
        <option
          key="2"
          value="01:15"
        >
          01:15
        </option>
        <option
          key="3"
          value="01:30"
        >
          01:30
        </option>
        <option
          key="4"
          value="01:45"
        >
          01:45
        </option>
        <option
          key="2"
          value="02:00"
        >
          02:00
        </option>
        <option
          key="3"
          value="02:15"
        >
          02:15
        </option>
        <option
          key="4"
          value="02:30"
        >
          02:30
        </option>
        <option
          key="5"
          value="02:45"
        >
          02:45
        </option>
        <option
          key="3"
          value="03:00"
        >
          03:00
        </option>
        <option
          key="4"
          value="03:15"
        >
          03:15
        </option>
        <option
          key="5"
          value="03:30"
        >
          03:30
        </option>
        <option
          key="6"
          value="03:45"
        >
          03:45
        </option>
        <option
          key="4"
          value="04:00"
        >
          04:00
        </option>
        <option
          key="5"
          value="04:15"
        >
          04:15
        </option>
        <option
          key="6"
          value="04:30"
        >
          04:30
        </option>
        <option
          key="7"
          value="04:45"
        >
          04:45
        </option>
        <option
          key="5"
          value="05:00"
        >
          05:00
        </option>
        <option
          key="6"
          value="05:15"
        >
          05:15
        </option>
        <option
          key="7"
          value="05:30"
        >
          05:30
        </option>
        <option
          key="8"
          value="05:45"
        >
          05:45
        </option>
        <option
          key="6"
          value="06:00"
        >
          06:00
        </option>
        <option
          key="7"
          value="06:15"
        >
          06:15
        </option>
        <option
          key="8"
          value="06:30"
        >
          06:30
        </option>
        <option
          key="9"
          value="06:45"
        >
          06:45
        </option>
        <option
          key="7"
          value="07:00"
        >
          07:00
        </option>
        <option
          key="8"
          value="07:15"
        >
          07:15
        </option>
        <option
          key="9"
          value="07:30"
        >
          07:30
        </option>
        <option
          key="10"
          value="07:45"
        >
          07:45
        </option>
        <option
          key="8"
          value="08:00"
        >
          08:00
        </option>
        <option
          key="9"
          value="08:15"
        >
          08:15
        </option>
        <option
          key="10"
          value="08:30"
        >
          08:30
        </option>
        <option
          key="11"
          value="08:45"
        >
          08:45
        </option>
        <option
          key="9"
          value="09:00"
        >
          09:00
        </option>
        <option
          key="10"
          value="09:15"
        >
          09:15
        </option>
        <option
          key="11"
          value="09:30"
        >
          09:30
        </option>
        <option
          key="12"
          value="09:45"
        >
          09:45
        </option>
        <option
          key="10"
          value="10:00"
        >
          10:00
        </option>
        <option
          key="11"
          value="10:15"
        >
          10:15
        </option>
        <option
          key="12"
          value="10:30"
        >
          10:30
        </option>
        <option
          key="13"
          value="10:45"
        >
          10:45
        </option>
        <option
          key="11"
          value="11:00"
        >
          11:00
        </option>
        <option
          key="12"
          value="11:15"
        >
          11:15
        </option>
        <option
          key="13"
          value="11:30"
        >
          11:30
        </option>
        <option
          key="14"
          value="11:45"
        >
          11:45
        </option>
        <option
          key="12"
          value="12:00"
        >
          12:00
        </option>
        <option
          key="13"
          value="12:15"
        >
          12:15
        </option>
        <option
          key="14"
          value="12:30"
        >
          12:30
        </option>
        <option
          key="15"
          value="12:45"
        >
          12:45
        </option>
        <option
          key="13"
          value="13:00"
        >
          13:00
        </option>
        <option
          key="14"
          value="13:15"
        >
          13:15
        </option>
        <option
          key="15"
          value="13:30"
        >
          13:30
        </option>
        <option
          key="16"
          value="13:45"
        >
          13:45
        </option>
        <option
          key="14"
          value="14:00"
        >
          14:00
        </option>
        <option
          key="15"
          value="14:15"
        >
          14:15
        </option>
        <option
          key="16"
          value="14:30"
        >
          14:30
        </option>
        <option
          key="17"
          value="14:45"
        >
          14:45
        </option>
        <option
          key="15"
          value="15:00"
        >
          15:00
        </option>
        <option
          key="16"
          value="15:15"
        >
          15:15
        </option>
        <option
          key="17"
          value="15:30"
        >
          15:30
        </option>
        <option
          key="18"
          value="15:45"
        >
          15:45
        </option>
        <option
          key="16"
          value="16:00"
        >
          16:00
        </option>
        <option
          key="17"
          value="16:15"
        >
          16:15
        </option>
        <option
          key="18"
          value="16:30"
        >
          16:30
        </option>
        <option
          key="19"
          value="16:45"
        >
          16:45
        </option>
        <option
          key="17"
          value="17:00"
        >
          17:00
        </option>
        <option
          key="18"
          value="17:15"
        >
          17:15
        </option>
        <option
          key="19"
          value="17:30"
        >
          17:30
        </option>
        <option
          key="20"
          value="17:45"
        >
          17:45
        </option>
        <option
          key="18"
          value="18:00"
        >
          18:00
        </option>
        <option
          key="19"
          value="18:15"
        >
          18:15
        </option>
        <option
          key="20"
          value="18:30"
        >
          18:30
        </option>
        <option
          key="21"
          value="18:45"
        >
          18:45
        </option>
        <option
          key="19"
          value="19:00"
        >
          19:00
        </option>
        <option
          key="20"
          value="19:15"
        >
          19:15
        </option>
        <option
          key="21"
          value="19:30"
        >
          19:30
        </option>
        <option
          key="22"
          value="19:45"
        >
          19:45
        </option>
        <option
          key="20"
          value="20:00"
        >
          20:00
        </option>
        <option
          key="21"
          value="20:15"
        >
          20:15
        </option>
        <option
          key="22"
          value="20:30"
        >
          20:30
        </option>
        <option
          key="23"
          value="20:45"
        >
          20:45
        </option>
        <option
          key="21"
          value="21:00"
        >
          21:00
        </option>
        <option
          key="22"
          value="21:15"
        >
          21:15
        </option>
        <option
          key="23"
          value="21:30"
        >
          21:30
        </option>
        <option
          key="24"
          value="21:45"
        >
          21:45
        </option>
        <option
          key="22"
          value="22:00"
        >
          22:00
        </option>
        <option
          key="23"
          value="22:15"
        >
          22:15
        </option>
        <option
          key="24"
          value="22:30"
        >
          22:30
        </option>
        <option
          key="25"
          value="22:45"
        >
          22:45
        </option>
        <option
          key="23"
          value="23:00"
        >
          23:00
        </option>
        <option
          key="24"
          value="23:15"
        >
          23:15
        </option>
        <option
          key="25"
          value="23:30"
        >
          23:30
        </option>
        <option
          key="26"
          value="23:45"
        >
          23:45
        </option>
        <option
          key="24"
          value="24:00"
        >
          24:00
        </option>
      </select>
      <input
        className="date-event end"
        name="endDateEvent"
        onChange={[Function]}
        type="date"
      />
    </div>
    <div
      className="description "
    >
      <i
        className="Tiny material-icons "
      >
        format_align_left
      </i>
      <textarea
        className="description-input "
        cols="42 "
        name="descriptionEvent"
        onChange={[Function]}
        placeholder="Add description "
        rows="4 "
      />
    </div>
    <div
      className="control "
    >
      <button
        className="submit-button "
        type="submit"
      >
        Save
      </button>
    </div>
  </form>
</Fragment>
`;
